internal/filesystem.go

```diff
@@ -420,14 +420,19 @@ func DBUSSessionBusAddress(id int64) string {
 	// #nosec G204 -- input is properly sanitized
 	out, err := exec.Command("ps", "-u", fmt.Sprintf("%d", id), "-o", "pid=").CombinedOutput()
 	if err != nil {
+		log.Println("Listing processes for uid: ", err.Error())
```

Comment:
Callers ignores errors anyways, I have just added logs for some utility.

Reply:
Generally it is good idea to give more leverage to the caller. Also, this may produce N logs every time daemon tries to send a notification for the user in case of permission denied accesss to `/proc/<pid>/environ` where N is pids per user.